src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { Tab } from '../../types/Tab';
+
+const tabs: Tab[] = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
+
+export const Tabs: React.FC = () => {
+  const { tabId } = useParams();
+
+  const selectedTab = tabs.find(tab => tab.id === tabId);
+
+  return (
+    <>
```

Comment:
Use some semantic element instead of react fragment. You have container for this content

```suggestion
   <div className="container">
```